tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -1840,6 +1851,58 @@ class GFF3Importer extends TripalImporter {
     }
   }
 
+
+  /**
+   * SOFIA:
+   * UPDATES the feature records in Chado.
+   */
+  private function updateFeatureNames() {
+    $batch_size = 1000;
+    $num_features = count(array_keys($this->update_names));
+    $num_batches = (int) ($num_features / $batch_size) + 1;
+
+    $this->setItemsHandled(0);
+    $this->setTotalItems($num_batches);
+#batch update: https://www.alibabacloud.com/blog/how-does-postgresql-implement-batch-update-deletion-and-insertion_596030
```

Comment:
indentation and change to two forward slashes.

Reply:
Fixed.